vendor/github.com/hashicorp/go-getter/client.go

```diff
@@ -289,7 +298,7 @@ func (c *Client) Get() error {
 		// if we're specifying a subdir.
 		err := g.Get(dst, u)
 		if err != nil {
-			err = fmt.Errorf("error downloading '%s': %s", src, err)
+			err = fmt.Errorf("error downloading '%s': %s", u.Redacted(), err)
```

Comment:
FWIW, this line is particularly interesting (the optional insecure config aside). 🤔 

My thinking being that this method would help certain, but not all cases where credentials are included as part of the URL. Not totally sure how I feel about it, but not a reason to block the PR.

Reply:
It might be nice to introduce in other use cases in go-getter itself, for sure.